src/main/groovy/org/sharegov/mdcgis/AddressService.groovy

```diff
@@ -44,6 +44,35 @@ class AddressService {
 	CacheManager cacheManager
 	GisConfig gisConfig
 
+	/**
+	 * get X and Y coordinates of address.
```

Comment:
you want to explain this better.
You want to explain that if more of one candidate you are returning a null